### PR TITLE
[docs sprint] Updates docs for react quickstart

### PR DESCRIPTION
**Pull request by ajar98**

### CODE REVIEW

File: docs/open-source/react-quickstart.mdx [hunk 1]
```diff
@@ -57,34 +57,22 @@ uvicorn main:app --port 3000
...
-```
-$ git clone https://github.com/vocodedev/vocode-react-demo.git
-```
-
-Run npm install inside the directory to download all of the dependencies.
-
-```
-$ npm install
 ```
+<>
```
Comment (adnaans) [inline docs/open-source/react-quickstart.mdx:71]: can we add js formatting here?

Comment (adnaans) [reply docs/open-source/react-quickstart.mdx:71]: or html? however we can format react nicely

File: docs/open-source/react-quickstart.mdx [hunk 2]
```diff
@@ -57,34 +57,22 @@ uvicorn main:app --port 3000
...
-```
-REACT_APP_VOCODE_API_KEY=YOUR KEY HERE
-```
-
-Start the application
-
-```
-$ npm start
+<button
+  disabled={["connecting"].includes(status)}
```
Comment (adnaans) [inline docs/open-source/react-quickstart.mdx]: what is "connecting" here?

**Review by adnaans — COMMENTED** [review]
few questions

File: docs/open-source/react-quickstart.mdx (hunk 2 above)
Comment (ajar98) [reply docs/open-source/react-quickstart.mdx]: it's an interim status exposed by the react-sdk: https://github.com/vocodedev/vocode-react-sdk/blob/main/src/types/conversation.ts#L6

i'm going to add something here that shows how to install the react-sdk

File: docs/open-source/react-quickstart.mdx (hunk 1 above)
Comment (ajar98) [reply docs/open-source/react-quickstart.mdx:71]: for future readers, mintlify uses prism for syntax highlighting: https://prismjs.com/#supported-languages

so I added `jsx` highlighting here

**Review by adnaans — APPROVED** [review]

**Pull request merged**